homeassistant/components/time/__init__.py

```diff
@@ -65,21 +71,24 @@ class TimeEntityDescription(EntityDescription, frozen_or_thawed=True):
     """A class that describes time entities."""
 
 
-class TimeEntity(Entity):
+CACHED_PROPERTIES_WITH_ATTR_ = {"native_value"}
```

Comment:
`device_class` and `state_attributes` are omitted here because they don't need a setter